samples/node/connector.js

```diff
@@ -24,16 +24,23 @@ app.use(bodyParser.urlencoded());
 app.set('trust proxy', true);
 app.use(['/cards/requests', '/reports'], function (req, res, next) {
     const authorization = req.header("authorization");
+    const xAuthorization = req.header("X-Connector-Authorization");
     if (authorization) {
         console.log(`Client passed "${authorization}". We should authenticate using a public key from VMware Identity Manager`);
-        next();
     } else {
         res.status(401).send("Missing Authorization header");
+        return
```

Comment:
You can just return the `return res.status(401).send('...');` (express doesn't care what is returned).